frontend/document/koptinterface.lua

```diff
@@ -381,6 +390,10 @@ function KoptInterface:renderOptimizedPage(doc, pageno, rect, zoom, rotation, re
 
     local cached = DocCache:check(hash, TileCacheItem)
     if not cached then
+        if hinting then
+            Device:enableCPUCores(2)
```

Comment:
Is that all that's actually useful in theory or just all that's actually useful in current practice?

Reply:
It's already barely useful here; as we're heavily single-threaded ;p.

It's mainly for the sake of the eInk driver itself, which is fairly terrible (read: resource intensive ;p).